source/reference/mongosync.txt

```diff
@@ -176,6 +176,38 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-oidc:
+
+Microsoft Azure Managed Identities Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/mongosync-and-oidc.rst
+
+To use Microsoft Azure Managed Identities for federated authentication,
+define these environment variables:
+
+.. option:: AZURE_TENANT_ID <id>
+
+   Sets the Azure tenant identifier. Example:
```

Comment:
Were I a user, I think I would expect some indication of how I can find my Azure tenant identifier. Can we add that detail, or link to some external doc that provides it?

Reply:
I dunno, I'm used to using a search engine for these things. https://learn.microsoft.com/en-us/entra/fundamentals/how-to-find-tenant

I'm not sure we should be in the business of documenting Microsoft's behavior